pkg/scheduler/framework/types.go

```diff
@@ -713,10 +715,23 @@ func max(a, b int64) int64 {
 
 // resourceRequest = max(sum(podSpec.Containers), podSpec.InitContainers) + overHead
 func calculateResource(pod *v1.Pod) (res Resource, non0CPU int64, non0Mem int64) {
```

Comment:
Do we need to update [`computePodResourceRequest`](https://github.com/kubernetes/kubernetes/blob/108c284a330a82ce1a1f80238e4f54bf5e8b045a/pkg/scheduler/framework/plugins/noderesources/fit.go#L162:6) as well?

ref #106769

Reply:
From what I can tell, we **do not** consider ResizeStatus for computePodResourceRequest. It needs to evaluate whether pod fits node based on **desired** Requests  - whether it is initial (i.e. scheduler is checking a candidate node) or if it is during resize (kubelet is checking whether the new desired Requests will fit)

@Huang-Wei @ahg-g I would really appreciate a second pair of eyes on my take about this.